src/share/poudriere/common.sh

```diff
@@ -7698,11 +7699,14 @@ build_repo() {
 	    err 1 "Unable to extract pkg."
 	run_hook pkgrepo sign "${PACKAGES}" "${PKG_REPO_SIGNING_KEY}" \
 	    "${PKG_REPO_FROM_HOST:-no}" "${PKG_REPO_META_FILE}"
+	PKG_META="-m /tmp/pkgmeta"
+	PKG_META_MASTERMNT="-m ${MASTERMNT}/tmp/pkgmeta"
 	if [ -r "${PKG_REPO_META_FILE:-/nonexistent}" ]; then
-		PKG_META="-m /tmp/pkgmeta"
-		PKG_META_MASTERMNT="-m ${MASTERMNT}/tmp/pkgmeta"
 		install -m 0400 "${PKG_REPO_META_FILE}" \
 		    ${MASTERMNT}/tmp/pkgmeta
+	else
+		printf "version = 2;\npacking_format = \"${PKG_EXT}\";\n" > \
```

Comment:
@bapt is there any risk to hardcoding this version here or in Makefile.inc1? It's backwards compatible?

Reply:
Now a moot point with pkg 1.17 accepting mixed compression formats